src/screens/post/container/postContainer.js

```diff
@@ -45,11 +49,10 @@ class PostContainer extends Component {
 
   componentWillReceiveProps(nextProps) {
     const { navigation } = this.props;
-    const { isFetch: nextIsFetch } =
-      nextProps.navigation.state && nextProps.navigation.state.params;
+    const { isFetch: nextIsFetch } = get(nextProps, 'navigation.state.params');
 
     if (nextIsFetch) {
-      const { author, permlink } = navigation.state && navigation.state.params;
+      const { author, permlink } = get(navigation, 'state.params');
```

Comment:
I think in here we should user like get(navigation.state, 'params'); because in prev line we are checking state value not navigation

Reply:
same thing